index.js

```diff
@@ -44,7 +44,7 @@ types[VISA] = {
   prefixPattern: /^4$/,
   exactPattern: /^4\d*$/,
   gaps: [4, 8, 12],
```

Comment:
Good find. I don't see anything in that document about 18 digits. Should we just remove the 18 digits from the array?